### ISSUE TITLE
Tokei 12.0.0 panics in u8::trim_first_and_last_line_of_whitespace

### ISSUE THREAD

**Issue by Ltrlg** [XAMPPRocky/tokei#566]
Just wanted to see the new `tokei` in action and ran it on our code base… to see it panic. Seems to be caused by a copy of https://github.com/mde/ejs/blob/v2.4.2/README.md that happens to be there, and more specifically by that part of the document:

```md
   ### Example
   
   ```html
  <div id="output"></div>
  <script src="ejs.min.js"></script>
  <script>
    var people = ['geddy', 'neil', 'alex'],
        html = ejs.render('<%= people.join(", "); %>', {people: people});
    // With jQuery:
    $('#output').html(html);
    // Vanilla JS:
    document.getElementById('output').innerHTML = html;
  </script>
  \```
```
(Backslash before backticks added for rendering, don’t know how to escape it properly.)

Backtrace:

```
thread '<unnamed>' panicked at 'index 1 out of range for slice of length 0', /mnt/c/Users/Lucien/.cargo/registry/src/github.co
m-1ecc6299db9ec823/tokei-12.0.0/src/utils/ext.rs:51:10
stack backtrace:
  [snip]
  13: core::slice::slice_index_len_fail
             at src/libcore/slice/mod.rs:2751
  14: <[u8] as tokei::utils::ext::SliceExt>::trim_first_and_last_line_of_whitespace
  15: tokei::language::syntax::SyntaxCounter::perform_multi_line_analysis
  16: tokei::language::language_type::LanguageType::parse_lines
  17: tokei::language::language_type::LanguageType::parse_from_slice
  18: tokei::language::syntax::SyntaxCounter::perform_multi_line_analysis
  19: tokei::language::language_type::LanguageType::parse_lines
  20: rayon_core::join::join_context::{{closure}}
  21: tokei::language::language_type::LanguageType::parse
  22: <rayon::iter::for_each::ForEachConsumer<F> as rayon::iter::plumbing::Folder<T>>::consume
  23: <rayon::iter::par_bridge::IterParallelProducer<Iter> as rayon::iter::plumbing::UnindexedProducer>::fold_with
  24: rayon::iter::plumbing::bridge_unindexed_producer_consumer
  25: <rayon_core::job::StackJob<L,F,R> as rayon_core::job::Job>::execute
  26: rayon_core::registry::WorkerThread::wait_until_cold
  27: rayon_core::registry::ThreadBuilder::run                                                                                
```

Happens with `default` features and `all-features`. Running on WSL1+Ubuntu18.04.4.

**Comment by XAMPPRocky** [comment]
Thank you for your issue! I've published 12.0.1 with a fix for this.

**Comment by Ltrlg** [comment]
Thanks!
